### DISCUSSION TITLE
I can't find my query

### DISCUSSION THREAD

**Discussion by noelia-fernandez-pe** [community/community#27633] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello everybody! A few days ago, on 07/18 I made a query which I titled ' github pages and repository files' (it assigned me the following number: 262365/3) but now I can't find it. I got the notification that someone answered me but the link in the email takes me to the Github Community home page. Where can I see my notifications? Where can I see what I have been answered?

**Comment by ihnatenkoo** [comment, marked as answer]
Hello, Noelia! Your requests can be found here (**Your Discussions - Created**)

![2022-07-29 20 56 35](https://user-images.githubusercontent.com/89586345/181817830-89aa0965-2eba-4958-b5df-857d9716f423.png)

**Reply by noelia-fernandez-pe** [reply]
Yes, but the query I made on 07/18 does not appear where you indicate. When I did it the community had another format.
In the 'created' section, only this last one appears.

**Comment by airtower-luna** [comment]
If your question hadn't been marked as solved yet, it's probably gone. :confused: One [answer in the recent AMA](https://github.com/orgs/community/discussions/27346#discussioncomment-3275305) was:

> we made the decision to only migrate over solutions from the Discourse forum, not all posts.

So all threads without a post marked a solution are gone. I guess the previous answer went in another direction because you used the term "request", like the support request page.

**Reply by noelia-fernandez-pe** [reply]
ok, thank you very much for the answer
